### TESTE T-STUDENT PARA UMA AMOSTRA - ONE-SAMPLE T TEST

Comparar a média de um grupo ou amostra contra um valor hipotético.

#### PREMISSAS:

-  Se n < 30, trata-se de uma amostra que não é grande o suficiente (menor de 30, teorema do limite central). Necessário verificar se os dados seguem uma distribuição normal.

#### HIPÓTESES:

- Ho: médias estatisticamente iguais
- Ha: médias estatisticamente diferentes

#### CRITÉRIOS:

- Se p-value for menor que nível de significância, rejeita-se Ho.

In [1]:
# DESATIVA NOTAÇÃO CIENTÍFICA
options(scipen = 999)

In [2]:
# DESATIVAR MENSAGENS DE WARNINGS (warn = -1) E ATIVAR (warn = 0)
options(warn = -1)

In [3]:
# CARREGAR BIBLIOTECAS
suppressPackageStartupMessages({
library(dplyr)
})

#### CARREGAR DATASET

Dataset contendo pesos de camundongos.

#### TESTE: 
Pode-se concluir que o peso médio dos camundongos é significativamente diferente de 25g?

In [4]:
load("data_rats.Rdata" )
dados_ratos <- as.data.frame(data_rats)
glimpse(dados_ratos)

Rows: 10
Columns: 2
$ name   <chr> "M_1", "M_2", "M_3", "M_4", "M_5", "M_6", "M_7", "M_8", "M_9", ~
$ weight <dbl> 20.3, 19.0, 19.1, 20.9, 18.6, 17.1, 21.1, 18.0, 20.0, 18.1


Amostra menor que 30. 

#### TESTE DE NORMALIDADE

In [5]:
shapiro.test(dados_ratos$weight)


	Shapiro-Wilk normality test

data:  dados_ratos$weight
W = 0.95989, p-value = 0.7846


#### TESTE T-STUDENT

In [6]:
t.test(dados_ratos$weight, mu = 25)


	One Sample t-test

data:  dados_ratos$weight
t = -13.788, df = 9, p-value = 0.000000234
alternative hypothesis: true mean is not equal to 25
95 percent confidence interval:
 18.27169 20.16831
sample estimates:
mean of x 
    19.22 


Conforme o Teste t-student acima, com um de p-value = 0.000000234, pode-se concluir que o peso dos camundongos é significamente diferente de 25g. Logo a hipótese nula (Ho) é rejeitada.